sdk/tests/types/address/bech32.rs

```diff
@@ -41,7 +41,7 @@ fn hrp_from_str() {
 
     assert!(matches!(
```

Comment:
Add an unpack test with the same invalid chars?

Reply:
Done